lblod/ember-submission-form-fields
# [LMB-34] feat: modelv2

transition to the final model v2:
- ~~move order to sections [todo]~~ this makes no sense, order is properly defined and works in the form builder as desired. I had misunderstood the extra indirection that was required by the form builder
- change propertygroups to sections [done]
- change sh:group to from:partOf [done]
- add new model example form [done]
- add rendersWhen constraint [done]

How to test:
- checkout this branch
- run the dummy application
- verify that the model for example form http://localhost:4200/dynamic-behaviour-v2 is indeed v2-target
- visit http://localhost:4200/dynamic-behaviour-v2 and verify that everything still works (new model)
- visit http://localhost:4200/dynamic-behaviour and verify that everything still works (old model)

![image](https://github.com/lblod/ember-submission-form-fields/assets/1076194/41fa4865-3222-467d-996d-0bb30a93ae15)

Would need an approve of https://github.com/lblod/submission-form-helpers/pull/55 to also support the new way of expressing validations.
